lib/usdUfe/ufe/Utils.cpp

```diff
@@ -116,10 +182,9 @@ void setUfePathToPrimFn(UfePathToPrimFn fn)
 
 UsdPrim ufePathToPrim(const Ufe::Path& path)
 {
-    if (nullptr == gUfePathToPrimFn) {
-        throw std::runtime_error("Path to prim function cannot be empty.");
-    }
-
+#if !defined(NDEBUG)
+    assert(gUfePathToPrimFn != nullptr);
+#endif
```

Comment:
As discussed I've removed the double nullptr check since for all these required DCC functions the check is done when the set is called (from initialize). So it should never be null here.